dev-db/etcd/files/etcd.service

```diff
@@ -6,6 +6,8 @@ User=etcd
 PermissionsStartOnly=true
 Environment=ETCD_DATA_DIR=/var/lib/etcd
 Environment=ETCD_NAME=%m
-ExecStart=/usr/bin/etcd 
+ExecStart=/usr/bin/etcd
 Restart=always
 RestartSec=10s
+LimitNOFILE=1048576
```

Comment:
Hm, which in turn we coppied from upstream docker which I'm sure picked the value out of thin air. Maybe something like `40000` would be a safer value so etcd leaking fds is less likely to harm the system as a whole. We should similarly reduce the value for flanneld.